mui/mui.py

```diff
@@ -234,6 +167,42 @@ def edit_custom_hook(bv: BinaryView, addr: int):
             bv.session_data.mui_custom_hooks[addr] = dialog.text()
 
 
+def load_evm(bv: BinaryView):
+    filename = get_open_filename_input("filename:", "*.sol").decode()
+    if filename is None:
+        return
+
+    print(filename)
```

Comment:
I think remove this or make it a more useful message. The rest of the print statements seem to make sense, but double-check to make sure they're all printing useful messages 👍

Reply:
Great point! I went back and reviewed all the print statements. Removed a few that are unnecessary.